tests/post_training/pipelines/base.py

```diff
@@ -500,9 +524,14 @@ def save_compressed_model(self) -> None:
             ov.serialize(ov_model, self.path_compressed_ir)
         elif self.backend in FX_BACKENDS:
             exported_model = torch.export.export(self.compressed_model.cpu(), (self.dummy_tensor.cpu(),))
-            ov_model = ov.convert_model(exported_model, example_input=self.dummy_tensor.cpu(), input=self.input_size)
-            ov_model.reshape(self.input_size)
-            ov.serialize(ov_model, self.path_compressed_ir)
+            torch.export.save(exported_model, self.output_model_dir / "model.pt2")
+            mod = torch.compile(
+                exported_model.module(),
+                backend="openvino",
+                options={"model_caching": True, "cache_dir": str(self.output_model_dir)},
+            )
+            mod(self.dummy_tensor)
+            self._rename_files(self.output_model_dir, "model")
```

Comment:
```suggestion
            self.path_compressed_ir = list(Path(self.output_model_dir / "model").glob("*.xml"))[0]
```
Just set path_compressed_ir to saved model

Reply:
Oh, this is clever. I will add it to the function since the function also raises error for graph breaks and OV not dumping the graph due to error in torch compile OV backend.

Reply:
Done